tests/unit/Dns/Cname/CnameRecordTest.php

```diff
@@ -52,4 +56,19 @@ private function api(MockHandler $mock): Api
             new Client(['handler' => $handler])
         );
     }
+
+    private function mock()
```

Comment:
Can you put in the return type here and re-name the method to `mockResponse()`

Reply:
Sure